cassandra/cluster.py

```diff
@@ -372,6 +373,14 @@ def auth_provider(self, value):
     If set to :const:`None`, there will be no timeout for these queries.
     """
 
+    idle_heartbeat_interval = 30
```

Comment:
> WRT missing events, we can at least rely on the new connection to refresh topology when it comes up.

That's true, but since the control connection doesn't regularly execute queries, it will in general be down until the next heartbeat fails.  It might be worth using a different heartbeat period for the control connection.  It probably wouldn't need a config option, either, since "accidentally" closing the control conn shouldn't have much of an effect on the application.  Unfortunately, this would slightly complicate the current heartbeat design.

But, overall, I think this default value would be reasonable to change in a future release if we see that it's not optimal in practice, so getting it perfect right now isn't crucial.